src/digitalIdentity-frontend/src/app/services/upload-file.service.ts

```diff
@@ -0,0 +1,9 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UploadFileService {
+
+  constructor() { }
+}
```

Comment:
This file (= the whole upload-file.service) has no funcionality and should be deleted (the tests for it as well).